boot/bootutil/src/loader.c

```diff
@@ -2262,12 +2262,11 @@ boot_prepare_image_for_update(struct boot_loader_state *state,
                  * magic, so also run validation on the primary slot to be
                  * sure it's not OK.
                  */
-                 if (boot_check_header_erased(state, BOOT_PRIMARY_SLOT) == 0 ||
-                     boot_validate_slot(state, BOOT_PRIMARY_SLOT, bs) != 0) {
-                     if (boot_img_hdr(state,
-                          BOOT_SECONDARY_SLOT)->ih_magic == IMAGE_MAGIC &&
-                         boot_validate_slot(state, BOOT_SECONDARY_SLOT, bs) == 0)
-                     {
+                if (boot_check_header_erased(state, BOOT_PRIMARY_SLOT) == 0 ||
```

Comment:
Oops, this was just an indentation issue.

Reply:
It does look better, but should probably be a separate commit.

Reply:
It was separated.